packages/components/src/Icon/Icon.mdx

```diff
@@ -25,6 +25,27 @@ Use icons as intended. Provided icons have a specific, meaning. To avoid
 confusing users, it’s essential that each image be used in accordance with its
 meaning and recommended usage.
 
+## Design Guidelines
+
+When contributing a new icon, follow these guidelines:
+
+### Size and Weight
+
+- Design within a 20px grid on a 24px artboard
```

Comment:
I don't quite follow this. Do you mean the icon should be within a 20px container within a 24px canvas, effectively meaning that there will be a 2px buffer around it?

Reply:
Yep! I can make that more clear